docs/components/_index.md

```diff
@@ -9,9 +9,22 @@ aliases:
   - "/components/"
 ---
 
-A _component_ represents a physical piece of electrical or electromechanical hardware in a machine.
+Viam provides support for a wide variety of hardware.
+A _component_ represents a physical piece of hardware in your {{< glossary_tooltip term_id="machine" text="machine" >}}, and the software that directly supports that hardware.
+
+{{< imgproc src="/viam/machine-components.png" alt="Diagram with various components and services on a smart machine. This machine employs the vision, navigation, and data capture services, which run within viam-server on the machine's single board computer." resize="1000x" style="max-width:650px" >}}
+<br>
+
+For each category (or _{{< glossary_tooltip term_id="subtype" text="subtype" >}}_) of hardware, such as robotic arms, Viam has a [standardized API](/build/program/apis/#component-apis).
+Within each category, there are different _{{< glossary_tooltip term_id="model" text="models" >}}_ that provide software drivers to support different hardware.
```

Comment:
```suggestion
Viam groups hardware, such as robotic arms into (or _{{< glossary_tooltip term_id="subtype" text="subtype" >}}_).
For each subtype, Viam has a [standardized API](/build/program/apis/#component-apis).
Within each subtype, there are different _{{< glossary_tooltip term_id="model" text="models" >}}_ that provide software drivers that implement the standardized API to support different hardware.
```

Reply:
I think there's a typo in your first sentence--would you like the word "categories" and the parentheses to stay or go?
How about this? I really worry the word "subtype" needs explicit explanation if we're using it at all.
```suggestion
Viam groups categories of hardware into _{{< glossary_tooltip term_id="subtype" text="subtypes" >}}_ of component.
For example, `arm` is a subtype of component that encompasses all robotic arms.
For each subtype, Viam has a [standardized API](/build/program/apis/#component-apis).
Within each subtype, there are different _{{< glossary_tooltip term_id="model" text="models" >}}_ that implement the standardized API with software drivers to support different hardware.
```

Reply:
what you suggested looks good! Maybe we should just altogether avoid subtype here. It's not really needed at this point. So how about
```suggestion
Viam groups categories of hardware together.
By grouping components, Viam can provide [standardized APIs](/build/program/apis/#component-apis) for components.
```

Reply:
```suggestion
Viam groups categories of hardware together, with a [standardized API](/build/program/apis/#component-apis) for all models in each grouping.
```